test/lint/lint-includes.sh

```diff
@@ -108,4 +111,6 @@ if [[ ${QUOTE_SYNTAX_INCLUDES} != "" ]]; then
     EXIT_CODE=1
 fi
 
+cd - > /dev/null || exit 1
```

Comment:
Nit: You can do `cd ~-` to avoid echoing :-)

Reply:
Ohh, that's nice one, I didn't know about `cd ~-`! :)